### DISCUSSION TITLE
How to check for variable name format.

### DISCUSSION THREAD

**Discussion by ChrisTaylorDeveloper** [PHP-CS-Fixer/PHP-CS-Fixer#7891] [Q&A] [answered]
How can I check for variable name format e.g. camel case, pascal case, snake case etc?

**Comment by Wirone** [comment, marked as answer]
There's a [PR](https://github.com/PHP-CS-Fixer/PHP-CS-Fixer/pull/5097) that covers this (at least partially), but unfortunately it's still in the works.
